haechan45th/src/routes/postRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const postController = require('../controllers/postController');
+
+const router = express.Router();
+
+router.get('/', postController.getAllPost);
```

Comment:
```suggestion
router.get('', postController.getAllPost);
```
엔드포인트 끝에는 일반적으로 `/`가 작성되지 않도록 하는것이 REST한 api 엔드포인트를 작성하는 것입니다. 다른 api들도 모두 반영해주세요~